tests/Ballot/Ballot.ts

```diff
@@ -86,72 +110,115 @@ describe("Ballot", function () {
   });
 
   describe("when the voter interact with the vote function in the contract", function () {
-    // TODO
-    it("is not implemented", async function () {
-      throw new Error("Not implemented");
+    it("should has the right to vote", async function () {
```

Comment:
I think what you meant was:
```suggestion
    it("should not have the right to vote", async function () {
```

Reply:
It's true it was a typo.